### DISCUSSION TITLE
请问怎么接入GPTS里的应用？

### DISCUSSION THREAD

**Discussion by haoqicunzhang** [zhayujie/chatgpt-on-wechat#1876] [Q&A]
请问怎么接入GPTS里的应用？

**Comment by Antelope-E** [comment]
使用中转站或自己搭逆向，一般中转站的GPTS模型格式是gpt-4-gizmo-*，代表该GPTS编号，比如image generator这个GPTS官网链接为https://chatgpt.com/g/g-pmuQfob8d-image-generator，那么他的模型就设置为gpt-4-gizmo-g-pmuQfob8d
